README.md

```diff
@@ -54,7 +54,7 @@ In this section:
 The [connector protocol](docs/CONNECTOR_PROTOCOL.md) is document-based and building a connector is language- and tool-agnostic.
 To create an Elastic connector, you need to implement this protocol. At a high level, you need to create an application (the connector service) that can read and write to a specific document in Elasticsearch that represents the connector. This document "registers" the connector with Kibana, keeps the configuration that is made via the Kibana UI, as well as the scheduling configuration and the state information for the service. You need to update the connector state to keep up with the current status of the connector application and of the third-party data source. You also need to log sync jobs to an additional index, so that the history of synchronization tasks would be available. And of course, you need to write the results of the synchronization to the Elasticsearch document index, created for this connector service.
 
-The procedure to properly implement the protocol _without the connector framework_ is as follows:
+The procedures to properly implement the protocol _without the connector framework_ is as follows:
```

Comment:
```suggestion
The procedure to properly implement the protocol _without the connector framework_ is as follows:
```

Reply:
If the verb is in single mode, then the subject should be in single mode.

Reply:
Then it should be changed to
>The procedures to properly implement the protocol _without the connector framework_ are as follows

WDYT?